nemo/collections/multimodal/speech_llm/modules/modality_adapters.py

```diff
@@ -132,3 +132,15 @@ def forward(self, audio_signal, length=None):
         outputs = self.mlp(outputs)
         outputs_len = torch.div(length, self.pooling_factor, rounding_mode='floor')
         return outputs.transpose(1, 2), outputs_len
+
+
+class IdentityConnectors(NeuralModule, Exportable, AccessMixin):
+    def __init__(
+        self,
+        *args,
```

Comment:
Unless it's necessary, remove *args and **,kwargs

Reply:
this part is necessary since to keep the same set of params when initializing the different types of connectors

Reply:
Sounds reasonable